app/views/catalog/_view_type_group.html.erb

```diff
@@ -1,5 +1,13 @@
-<%= render(ViewTypeDropdownComponent.new(
```

Comment:
Can you remove the ViewTypeDropdownComponent and ViewTypeDropdownItemComponent in here too?